### PR TITLE
Setup `pipenv` for package management in Django

### PR DESCRIPTION
**Pull request by matthew-t-smith**
- Installed previous dependencies via pipenv to `Pipefile`
- Remove requirements.txt
- Updated README.md with new instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,21 +7,43 @@ data from `localhost:8000` **where the Django app must be running**.
...
+
+### Installing new packages
+- Simply install via: `pipenv install [package-name]`
+
+### Create dotenv file with app secret
 - `echo "SECRET_KEY='TEMPORARY SECRET KEY'" > vp/.environment`
 
-**Start dev server from app root**
+### Start dev server from app root
 - `cd vp`
```
Comment (reelmatt) [inline README.md]: Missing `pipenv shell` step. Need to execute before `cd vp`, otherwise complains about missing modules.

File: Pipfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
```
Comment (reelmatt) [inline Pipfile]: We should specify package versions like we had in `requirements.txt`

File: README.md (hunk 1 above)
Comment (matthew-t-smith) [reply README.md]: `pipenv shell` should only be needed to spawn the virtualenv shell. Wouldn't be needed to install the requirements at the `visual-programming` level. What are the warnings you're getting? 

You can change directories around and still start up a Pipenv shell, too.

File: Pipfile (hunk 2 above)
Comment (matthew-t-smith) [reply Pipfile]: Updated with versions specified - allows for minor updates only.

**Comment by matthew-t-smith** [comment]
@cesaragv - Pipenv does recommend keeping the Pipfile.lock in version control unless you're needing multiple python versions.

File: README.md (hunk 1 above)
Comment (reelmatt) [reply README.md]: Sorry, I worded that poorly. Starting the server without the `pipenv shell` or `pipenv run` steps you added to the README produced a `ModuleNotFoundError`; I had only tried the `shell` option. All is good now 👍

**Pull request merged**